frontend/src/components/Footer/index.tsx

```diff
@@ -1,10 +1,44 @@
 import React, { FC } from "react";
-import { MainWrapper, Wrapper } from "./style";
+import { MainWrapper, StyledAnchor, Wrapper } from "./style";
 
 const Footer: FC = () => {
   return (
     <Wrapper>
-      <MainWrapper>© 2020 cellxgene</MainWrapper>
+      <MainWrapper>
+        © 2020 cellxgene
+        <StyledAnchor
+          key={"mailto:cellxgene@chanzuckerberg.com"}
```

Comment:
nit: oh we only need `key` when we're generating components in a `map()` call 😄

Reply:
Ahh oops! Deleted!